articles/client-auth/v2/mobile-desktop.md

```diff
@@ -0,0 +1,444 @@
+---
+title: Authentication for Mobile & Desktop Apps
+description: Explains how to authenticate users in a mobile or desktop application.
+toc: true
+---
+
+# Authentication for Mobile & Desktop Apps
+
+You can authenticate users of your mobile/desktop applications by:
+
+* Using the [Lock](/libraries/lock) client libraries;
+* Calling the Auth0 OAuth 2.0 endpoints.
+
+This article will cover how to call the Auth0 OAuch 2.0 endpoints using [Proof Key for Code Exchange (PKCE)](/api-auth/grant/authorization-code-pkce) during the authentication process.
```

Comment:
Typo `Auth0 OAuch 2.0`. Better to replace it with `[Auth0 Authentication API](/api/authentication) endpoints`.